tools/util.py

```diff
@@ -11,10 +11,12 @@
 import time
 import unittest
 
-# FIXME support nocolor (use "" if passed?)
-RESET = "\x1b[0m"
-FG_RED = "\x1b[31m"
-FG_GREEN = "\x1b[32m"
+if os.environ.get("NO_COLOR", None):
+    RESET = FG_READ = FG_GREEN = ""
```

Comment:
This still doesn't work on appveyor, is that an appveyor bug or a Windows one? (should we set NO_COLOR on appveyor?)

https://ci.appveyor.com/project/deno/deno/builds/24934612

Reply:
Not sure what's the problem here, can you explain?

Reply:
The "ok" is on a new line, unlike on max/linux:
```
032_xeval_delim (integration_tests.TestIntegrations) ... 
ok
async_error (integration_tests.TestIntegrations) ... 
ok
circular1 (integration_tests.TestIntegrations) ... 
ok
config (integration_tests.TestIntegrations) ... 
ok
```
This is kind of an aside, but was the impetus for me making the NO_COLOR comment!